zaptest/observer/observer.go

```diff
@@ -78,23 +78,30 @@ func (o *ObservedLogs) AllUntimed() []LoggedEntry {
 	return ret
 }
 
+// FilterLevel filters entries to those logged at exactly the given level.
```

Comment:
optional: In other places in zap, level is the minimum level but it's an exact match.

To avoid confusion, should this be include "Exact" or "Same" in the name, `FilterLevelExact`?

Reply:
👍 
I can imagine a future `FilterLevelAbove` or `FilterLevelAtLeast`.